packages/main/cypress/specs/Input.cy.tsx

```diff
@@ -625,51 +614,54 @@ describe("Input PAGEUP/PAGEDOWN navigation", () => {
 describe("Selection-change event", () => {
 	it("Selection-change event fires with null arguments when suggestion was selected but user alters input value to something else", () => {
 		cy.mount(
-			<Input id="input-selection-change" showSuggestions>
+			<Input showSuggestions>
 				<SuggestionItem text="Cozy" />
 				<SuggestionItem text="Compact" />
 				<SuggestionItem text="Condensed" />
 			</Input>
 		);
 
-		cy.get("ui5-input")
-			.as("input");
-
-		cy.get("ui5-input")
-			.shadow()
-			.find("input")
-			.as("innerInput");
-
 		let eventCount = 0;
 
-		cy.get("@input").then($input => {
-			$input[0].addEventListener("ui5-selection-change", () => {
-				eventCount++;
+		cy.get("ui5-input")
+			.as("input")
+			.then($input => {
+				$input[0].addEventListener("ui5-selection-change", () => {
+					eventCount++;
+				});
```

Comment:
Can we move this to the mount: <Input showSuggestions onSelectionChange={cy.stub().as("inputSelectionChange")> ?

Reply:
Done